src/QMCWaveFunctions/tests/test_RotatedSPOs.cpp

```diff
@@ -523,7 +523,9 @@ TEST_CASE("RotatedSPOs hcpBe", "[wavefunction]")
 
   const char* particles = R"(<tmp>
 <sposet_builder type="bspline" href="hcpBe.pwscf.h5" tilematrix="1 0 0 0 1 0 0 0 1" twistnum="0" source="ion" meshfactor="1.0" precision="double" size="2">
-    <sposet type="bspline" name="spo_ud" size="2" spindataset="0" optimize="yes"/>
+    <rotated_sposet>
```

Comment:
The above `size` input needs to be removed.

Reply:
If the size on "sposet_builder" is removed, the code halts in EinsplineSetBuilder_createSPOSet::set_metadata with an error ("You must specify the number of orbitals in the input file").
However, set_metadata doesn't seem to do anything with the number of orbitals other than check it and issue the error message.

Reply:
Did you specify size on the sposet line?
See a valid input
https://github.com/QMCPACK/qmcpack/blob/95c4640e42ea91dd04e431f7a74a85ad1fbe6874/src/QMCWaveFunctions/tests/test_spo_collection_input_spline.cpp#L144
Probably I should add a trap to block "size" on the sposet_collection/builder line

Reply:
I fixed the call to EinsplineSetBuilder::createSPOSetFromXML to better match the calling sequence in the code.  Now setting "size" works as expected on the right element.

Reply:
My point was that size is a parameter of a given sposet not the builder.